samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -1502,6 +1509,47 @@ void CEncodingPipeline::InsertIDR(bool bIsNextFrameIDR)
     }
 }
 
+mfxStatus CEncodingPipeline::InitEncFrameParams(sTask* pTask)
+{
+    MSDK_CHECK_POINTER(pTask, MFX_ERR_NULL_PTR);
+
+    mfxStatus sts = MFX_ERR_NONE;
+
+    pTask->extBufs = m_encExtBufs.GetFreeSet();
```

Comment:
generally it is OK to not have any ext buffers. For FEI such situation was impossible, since FEI has some mandatory run time buffers, but legacy encode doesn't

Reply:
OK, will check it in the beginning.